Resources/Prototypes/Roles/MindRoles/mind_roles.yml

```
@@ -113,18 +113,19 @@
   - type: MindRole
     antagPrototype: SubvertedSilicon
     roleType: SiliconAntagonist
+    subtype: "rule-subtype-subverted"
```

```suggestion
    subtype: rule-subtype-subverted
```
these don't need to be in speech marks I think
same for the others below